src/actions/actions.js

```diff
@@ -1,11 +1,11 @@
-import { FETCH_MONSTERS, FETCH_ONE_MONSTER } from './types'
+import { FETCH_MONSTERS, FETCH_ONE_MONSTER, SEARCH_MONSTER_NAME } from './types'
 
 export const getMonsters = () => dispatch =>{
   fetch('https://www.dnd5eapi.co/api/monsters')
     .then(response => response.json())
     .then(monsters => dispatch({
       type: FETCH_MONSTERS,
-      payload: monsters
+      payload: monsters.results
```

Comment:
Good call on moving `.results` here closer to the API call so that we're not passing around extra data and having to append `.results` in other files.